doc/admin-api.md

```diff
@@ -70,7 +70,11 @@
 |upstream_id|False |Upstream|Enabled upstream id, see [Upstream](architecture-design.md#upstream) for more ||
 |service_id|False |Service|Binded Service configuration, see [Service](architecture-design.md#service) for more ||
 |service_protocol|False|Upstream protocol type|only `grpc` and `http` are supported|`http` is the default value; Must set `grpc` if using `gRPC proxy` or `gRPC transcode`|
+<<<<<<< HEAD
```

Comment:
it seems that you did not rebase your branch in the correct way.

please fix it.